src/lib/utils.py

```diff
@@ -166,7 +166,10 @@ def filter_versions(
             try:
                 match = oper(StrictVersion(version_str), StrictVersion(version_limit))
             except ValueError:
-                match = oper(LooseVersion(version_str), LooseVersion(version_limit))
+                try:
+                    match = oper(LooseVersion(version_str), LooseVersion(version_limit))
+                except TypeError:
```

Comment:
Please capture and debug-log the exception.

Reply:
Added the debug log.